packages/cli/install.sh

```diff
@@ -0,0 +1,54 @@
+#!/bin/sh
+
+set -e
+
+if [ "$OS" = "Windows_NT" ]; then
+	target="win32-x64"
+	ext="exe"
```

Comment:
why are you using tabs in this script?
why the windows assertion is needed here? isnt it for bash envs?

Reply:
there's bash for windows, and there are multiple ways to work with bash on windows:
- when installing git on windows -> bash is installed by default.
- "windows terminal"  (an official Microsoft terminal - https://apps.microsoft.com/detail/9n0dx20hk701?hl=en-US&gl=US) supports working with bash.
- WSL

Reply:
actually, from a WSL terminal - PowerShell will not work.